packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx

```diff
@@ -70,23 +69,20 @@ class OrderBookSide extends Component<OrderBookSideProps, {}> {
   static defaultProps = {
     fixedPrecision: 4,
     pricePrecision: 4,
-    scrollToTop: false,
   };
 
   componentDidMount() {
-    if (this.props.scrollToTop) this.side.scrollTop = this.side.scrollHeight;
+    const { type } = this.props;
+    this.side.scrollTop = type === BIDS ? 0 : this.side.scrollHeight;
```

Comment:
what about ask section of orderbook? is that not an issue?

Reply:
this covers both now instead of passing a prop -- so if it's bids scroll to the top of the side (best bid) if it's asks scroll to the bottom (scroll height)